rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -16,6 +16,8 @@ toc::[]
 [id="rosa-q3-2024_{context}"]
 === Q3 2024
 
+* **IMDSv2 support in {hcp-title}.** You can now configure the IMDSv2 endpoint for the default machine pool on new {hcp-title} clusters and new machine pools on existing clusters. For more information, see xref:..//rosa_hcp/terraform/rosa-hcp-creating-a-cluster-quickly-terraform.adoc#rosa-hcp-creating-a-cluster-quickly-terraform[Creating a default ROSA cluster using Terraform].
```

Comment:
It's not IMDSv2 endpoint that can now be configured. So, I suggest following change: 

_EC2 Instance Metadata Service (IMDS) Configuration: You can now enforce the use of IMDSv2 endpoint for default machine pool worker nodes on new HCP cluster and on new machine pools on existing clusters._

Reply:
Thank you! I updated the note

Reply:
I would disagree. The change is we are introducing is support to allow the use of the IMDSv2 endpoint. Before this release, you could not use IMDSv2, only IMDSv1. I would likely change the suggested wording to:
> * **Amazon EC2 Instance Metadata Service Version 2 (IMDSv2) Support** You can now require the use of the IMDSv2 endpoint for new machine pools, including the default machine pool in newly created clusters. Existing clusters must create a new machine pool to require the use of IMDSv2.

Reply:
I see enforce is a correct word that @bchandra-ocp  suggested, becuase if we configure imdsv2 to required, it cannot be after ward changed to optional, which means enforcing it

Reply:
@michaelryanmcneill Can you clarify why before this feature one could not use IMDSv2?

Reply:
@bchandra-ocp I clarified this in my other comment but will add a little more detail here. IMDSv2 is always "enabled" when the instance metadata service is enabled on an EC2 instance (OpenShift requires the instance metadata service to be enabled). Setting this feature to `optional` (which is the default) means that the EC2 instances _can use_ IMDSv1 or IMDSv2 for instance metadata. In OpenShift's case, this means that the instances **will never use** IMDSv2, and will only use the less-secure IMDSv1, because Ignition pulls from the IMDSv1 by default. Other applications which might need to pull instance metadata can use the IMDSv1 or IMDSv2 endpoint in `optional` mode. Setting this feature to `required` means that the EC2 instances use IMDSv2 for instance metadata and cannot use IMDSv1. This setting is enforced via an EC2 specific configuration which is managed by the Machine API. Hope that makes a little more clarity.